frontend/fingertips-frontend/eslint.config.mjs

```diff
@@ -1,13 +1,26 @@
-import path from "node:path";
-import { fileURLToPath } from "node:url";
-import js from "@eslint/js";
-import { FlatCompat } from "@eslint/eslintrc";
+import path from 'node:path';
+import { fileURLToPath } from 'node:url';
+import js from '@eslint/js';
+import { FlatCompat } from '@eslint/eslintrc';
 
 const __filename = fileURLToPath(import.meta.url);
 const __dirname = path.dirname(__filename);
 const compat = new FlatCompat({
-    baseDirectory: __dirname,
-    recommendedConfig: js.configs.recommended,
-    allConfig: js.configs.all
+  baseDirectory: __dirname,
+  recommendedConfig: js.configs.recommended,
+  allConfig: js.configs.all,
 });
-export default [...compat.extends("next/core-web-vitals", "next/typescript")];
\ No newline at end of file
+export default [
+  ...compat.extends('next/core-web-vitals', 'next/typescript'),
+  {
+    rules: {
+      '@typescript-eslint/no-namespace': 'off',
+      '@typescript-eslint/no-unused-vars': [
+        'error',
+        {
+          varsIgnorePattern: '^_',
+        },
+      ],
+    },
+  },
```

Comment:
I thought we saw that we didn't need the `error` entry @alibombaye?
```suggestion
    rules: {
      '@typescript-eslint/no-namespace': 'off',
      '@typescript-eslint/no-unused-vars': [
        {
          varsIgnorePattern: '^_',
        },
      ],
    },
  },
```

Reply:
Unfortunately, it caused a linting issue and I had to put it back in. 
```
Config (unnamed): Key "rules": Key "@typescript-eslint/no-unused-vars": Expected severity of "off", 0, "warn", 1, "error", or 2.
```
I've set it as error since I think we do want to flag up as errors if there are any unused vars we explicitly don't use the `_` prefix. Happy to change to a different level if we think it's sensible?

Reply:
No, that's fine. I hadn't particularly looked into what this config value was doing, but I agree we should flag unused variables without the prefix as errors.